app/controllers/api_resource_comments_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ApiResourceCommentsController < ApiApplicationController
+  def show
+    resource_id = params['resource_id']
+    resource = Resource.find(resource_id)
```

Comment:
This query is not needed. You can replace the next line by the following:
`comments = ResourceComment.where(resource_id:)`

Reply:
This query it's needed because the `.where()` method returns an empty array when the Resource doesn't exist. With this query, if the `resource_id` is invalid, it returns a 404 error, which I think it should be the desired behavior.

Reply:
Oh, you are right. Then I'd use a one-liner like this one: `Resource.find(resource_id)&.resource_comments` (Valentin's idea)

Reply:
But also not strictly necessary